mmdet3d/models/losses/axis_aligned_iou_loss.py

```diff
@@ -0,0 +1,88 @@
+import torch
+from torch import nn as nn
+
+from mmdet.models.builder import LOSSES
+from mmdet.models.losses.utils import weighted_loss
+
+
+@weighted_loss
+def axis_aligned_iou_loss(pred, target, eps=1e-6):
+    """Calculate the IoU loss (1-IoU) of two set of axis aligned bounding
+    boxes. Note that predictions and targets are one-to-one corresponded.
+
+    Args:
+        pred (torch.Tensor): Bbox predictions with shape [..., 3].
+        target (torch.Tensor): Bbox targets (gt) with shape [..., 3].
+
+    Returns:
+        torch.Tensor: IoU loss between predictions and targets.
+    """
+
+    (min_a, max_a) = torch.split(pred, 3, dim=-1)
```

Comment:
It seems that line 21-32 can be abstract as `axis_aligned_iou`, we can do that thus this IoU function can be used in more places.